cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -339,6 +374,95 @@ impl YaccParser {
         }
     }
 
+    fn parse_yacckind(&mut self, i: usize, update_yacc_kind: bool) -> Result<usize, YaccGrammarError> {
+        const YACC_KINDS: [(&str, YaccKind); 5] = [
+            ("Grmtools", YaccKind::Grmtools),
+            (
+                "Original(NoAction)",
+                YaccKind::Original(YaccOriginalActionKind::NoAction),
+            ),
+            (
+                "Original(UserAction)",
+                YaccKind::Original(YaccOriginalActionKind::UserAction),
+            ),
+            (
+                "Original(GenericParseTree)",
+                YaccKind::Original(YaccOriginalActionKind::GenericParseTree),
+            ),
+            ("Eco", YaccKind::Eco),
+        ];
+        let j = self.parse_ws(i, false)?;
+        let s = &self.src[i..];
+        for (kind_name, kind) in YACC_KINDS {
+            if s.starts_with(kind_name) {
+                if update_yacc_kind {
+                    self.yacc_kind = kind;
+                }
+                let end_pos = j + kind_name.len();
+                return Ok(end_pos);
+            }
+        }
+        // This strikes me as an interesting case where we know the start pos of the
+        // invalid yacc kind value, but do not know the end, we could `self.parse_to_eol()`.
+        Err(self.mk_error(YaccGrammarErrorKind::InvalidYaccKind, i))
+    }
+    fn parse_grmtools_header(&mut self, mut i: usize) -> Result<usize, YaccGrammarError> {
+        let mut yacc_kind_key_span = None;
+        let mut _yacc_kind_val_span = None;
```

Comment:
If we're not using this, we should probably remove the variable?

Reply:
Should be fixed in [a307980](https://github.com/softdevteam/grmtools/pull/491/commits/a307980095753e5fa028e6de465eab90a7fd7e72)